### PR TITLE
GEODE-8181: Fixes default value for statistic-sampling-enabled.

### PR DESCRIPTION
**Pull request by pivotal-jbarrett**
GEODE-8181: Fixes default value for statistic-sampling-enabled.

### CODE REVIEW

File: cppcache/integration-test/testSystemProperties.cpp [hunk 1]
```diff
@@ -51,7 +51,7 @@ BEGIN_TEST(DEFAULT)
     ASSERT(
         systemProperties->statisticsSampleInterval() == std::chrono::seconds(1),
         "expected 1");
-    ASSERT(systemProperties->statisticsEnabled() == true, "expected true");
+    ASSERT(systemProperties->statisticsEnabled() == false, "expected true");
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/testSystemProperties.cpp:54]: The string description should now be "expected false," no?

File: cppcache/integration-test/testSystemProperties.cpp [hunk 2]
```diff
@@ -80,7 +80,7 @@ BEGIN_TEST(NEW_CONFIG)
                std::chrono::seconds(700),
            "expected 700");
 
-    ASSERT(systemProperties->statisticsEnabled() == false, "expected false");
+    ASSERT(systemProperties->statisticsEnabled() == true, "expected false");
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/testSystemProperties.cpp:83]: Same here

File: cppcache/integration-test/resources/system.properties [hunk 3]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # All the configurable parameters.
 statistic-sample-rate=700s
-statistic-sampling-enabled=false
+statistic-sampling-enabled=true
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/resources/system.properties:17]: I don't understand this change.  Doesn't this change the value to true for _all_ integration tests now?  I thought only the expiration tests required this, i.e. the two places where you're now explicitly enabling them in test code.

File: cppcache/integration-test/testSystemProperties.cpp (hunk 1 above)
Comment (pivotal-jbarrett) [reply cppcache/integration-test/testSystemProperties.cpp:54]: Ha! Details!

File: cppcache/integration-test/resources/system.properties (hunk 3 above)
Comment (pivotal-jbarrett) [reply cppcache/integration-test/resources/system.properties:17]: No it’s used by a single integration test.

Comment (pdxcodemonkey) [reply cppcache/integration-test/resources/system.properties:17]: Ugh.  I'd ask that we rename this file to reflect that it's test-case-specific, but I think time is better spent just getting rid of the old tests in general.  Thanks for the clarification.

Comment (pivotal-jbarrett) [reply cppcache/integration-test/resources/system.properties:17]: Yeah, when I need more time in C++ again after ACE is gone...

**Pull request merged**